File: 1.py
# 冒泡算法
def bubble_sort(arry):
    n = len(arry)
    for i in range(n):
        for j in rang(1,n-1):
            if arry[j-1] > arry[j]:
                arry[j-1],arry[j] = arry[j],arry[j-1]
    return arry

File: 1.cs
//冒泡排序
public static void BubbleSort(int[] list)
{
  for (int i = list.length-1;i>0;i--)
  {
    for (int j = 0;j<i;j++)
    {
      if (list[j]>list[j+1])
      {
        int temp = list[j];
        list[j] = list[j+1];
        list[j+1] = temp;
      }
    }
  }
}
